opentech/public/projects/templates/projects/project_page.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% load wagtailcore_tags wagtailimages_tags %}
+
+
+{% block content %}
+    <div class="intro">
+        <div class="container">
+            {% if page.icon %}
+                {% image page.icon original %}
+            {% endif %}
+            <h1>{{ page.title }}</h1>
+            {% if page.introduction %}
+                <p>{{ page.introduction }}</p>
+            {% endif %}
+
+            {{ page.body }}
+
+            {% with contact_details=page.contact_details.all %}
+                {% if contact_details %}
+                    {% for contact in contact_details %}
+                <p><a href="{{ contact.url }}">
+                    <span class="icon {{ contact.service }}">{{ contact.service_name }}
+                        </a></p>
+                    {% endfor %}
+
+                {% endif %}
+            {% endwith %}
+
+        </div>
+    </div>
+    {% include "utils/includes/funding.html" %}
+
+    {% regroup page.category_options by category as categories %}
```

Comment:
So, the widget shows the options in the correct order when editing the project 👍 
The order is not preserved here. 👎

Reply:
`regroup` preserves order. I have updated the query to ensure the correct order is passed into the template